doc/source/conf.py

```diff
@@ -131,16 +132,45 @@
 
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+# https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/branding.html
 
-html_theme = "scikit-image"
-html_theme_path = ["themes"]
-html_title = f"skimage v{version} docs"
+html_theme = "pydata_sphinx_theme"
 html_favicon = "_static/favicon.ico"
 html_static_path = ["_static"]
+html_logo = "_static/logo.png"
+
+html_css_files = ['theme_overrides.css']
+
+html_theme_options = {
+    "github_url": "https://github.com/scikit-image/scikit-image",
+    "navbar_start": ["navbar-logo", "version-switcher"],
+    "logo": {
+        "alt_text": (
+            "scikit-image's logo, showing a snake's head overlayed with green "
+            "and orange"
+        ),
+        "text": "scikit-image"
+    },
+    "icon_links": [
+        {
+            "name": "PyPI",
+            "url": "https://pypi.org/project/scikit-image/",
+            "icon": "fa-solid fa-box",
+        },
+    ],
+    "switcher": {
+        # "json_url": "https://scikit-image.org/docs/dev/_static/version_switcher.json",
+        "json_url": "https://gist.githubusercontent.com/lagru/8bce79c510b9026dab455c0a96a2c0b3/raw/18fcf0c82af932ba3c13033e12bd6923b8f2034c/version_switcher.json",
```

Comment:
Update this before merge!

Reply:
We forgot to update this. :sweat_smile: See https://github.com/scikit-image/scikit-image/pull/6882 for the fix.

Reply:
Darn it, I *knew* that was going to bite us!